src/main/java/org/sonarlint/intellij/config/global/SonarQubeServerEditor.java

```diff
@@ -36,19 +36,22 @@
 import java.util.HashSet;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'java.util.HashSet'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)